.eslintrc.yml

```diff
@@ -0,0 +1,8 @@
+env:
+  browser: true
```

Comment:
Is this a server side only code? If yes then `browser` should be changed to `node`, like
https://github.com/app-sre/qontract-server/blob/7b24629c73704d0041cdb7885427f62dc9214115/.eslintrc.json#L6C5-L6C18